setup.py

```diff
@@ -31,7 +31,11 @@
 grpc_requirements = ["grpcio==1.27.2"]
 kubeflow_requirements = ["kubernetes", "minio", "google-cloud-storage", "sh"]
 media_requirements = ["numpy", "moviepy", "pillow", "bokeh", "soundfile", "plotly"]
-launch_requirements = ["docker", "jupyter-repo2docker", "chardet", "iso8601"]
+launch_requirements = [
+    "docker",
```

Comment:
We shouldn't need `docker` for anything.

Reply:
Oops, thought I got rid of that. Removing